resources/default.rb

```diff
@@ -4,3 +4,7 @@
 
 attribute :base_name, name_attribute: true, kind_of: String, required: true
 attribute :version, kind_of: String, required: true
+attribute :push_to_datadog, kind_of: [TrueClass, FalseClass], required: false
+attribute :datadog_event_name, kind_of: String, required: false
```

Comment:
add a `default: nil` to all your strings to get the code up there working correctly